blueprints/ember-cli-eslint/files/.eslintrc.js

```diff
@@ -1,3 +1,7 @@
+var path = require('path');
+
 module.exports = {
-  extends: './node_modules/ember-cli-eslint/coding-standard/ember-application.js'
+  extends: [
+    path.join(__dirname, 'node_modules/ember-cli-eslint/coding-standard/ember-application.js')
```

Comment:
I think it may be better to use `require.resolve('ember-cli-eslint/coding-standard/ember-application')`, can you check that?